kafka-daq-report/src/app.rs

```diff
@@ -1,7 +1,7 @@
 use chrono::{DateTime, Timelike, Utc};
 use ratatui::widgets::TableState;
 
-use super::SharedData;
+use super::CommonData;
```

Comment:
So, this wasn't the use of the term "shared" I meant, but this was also a bad name, the new name is not much better though.
Could the name here reflect what is actually stored, not just the fact it is shared.